src/dags/hoofdroutes_verkeer.py

```diff
@@ -0,0 +1,207 @@
+    """
+    tmp_dir = Path(f"/tmp/{dag_id}")
+    tmp_dir.mkdir(parents=True, exist_ok=True)
+
+    # 1. download files
+    files = {}
+    for route in ROUTES:
+        dest = f"{tmp_dir}/{route.name}.geojson"
+        logger.info("Downloading %s to %s", route.url, dest)
+        download_file(route.url, dest, http_conn_id=None)
+        files[route.name] = dest
+
+    # 2. generate schema ("schema introspect geojson *.geojson")
+    schema = introspect_geojson_files("gevaarlijke-routes", files=list(files.values()))
+    schema = DatasetSchema.from_dict(schema)  # TODO: move to schema-tools?
+
+    # XXX This is not running as one transaction atm, but autocommitting per chunk
+    # 3. import data
+    db_engine = get_engine()
+    importer = GeoJSONImporter(schema, db_engine, logger=logger)
+    for route in ROUTES:
+        geojson_path = files[route.name]
+        logger.info("Importing %s into %s", route.name, route.tmp_db_table_name)
+        importer.load_file(
+            geojson_path,
+            table_name=route.schema_table_name,
+            db_table_name=route.tmp_db_table_name,
+            truncate=True,  # when reexecuting the same task
+        )
+        if route.post_process:
+            hook = PostgresHook(postgres_conn_id=postgres_conn_id)
+            hook.run(route.post_process)
+
+
+with DAG(dag_id, default_args=default_args) as dag:
+
+    count_checks = []
+    colname_checks = []
+    geo_checks = []
+    renames = []
+
+    drop_old_tables = PostgresOperator(
+        task_id="drop_old_tables", sql=DROP_TMPL, params=dict(tablenames=TABLES_TO_DROP)
+    )
+
+    import_geojson = PythonOperator(
+        task_id="import_geojson",
+        python_callable=_load_geojson,
+        op_args=[default_args.get("postgres_conn_id", "postgres_default")],
+    )
+
+    for route in ROUTES:
+        count_checks.append(
+            COUNT_CHECK.make_check(
+                check_id=f"count_check_{route.name}",
+                pass_value=3,
+                params=dict(table_name=route.tmp_db_table_name),
+                result_checker=operator.ge,
+            )
+        )
+
+        colname_checks.append(
+            COLNAMES_CHECK.make_check(
+                check_id=f"colname_check_{route.name}",
+                parameters=[route.tmp_db_table_name],
+                pass_value=set(route.columns),
+                result_checker=operator.ge,
+            )
+        )
+
+        geo_checks.append(
+            GEO_CHECK.make_check(
+                check_id=f"geo_check_{route.name}",
+                params=dict(
+                    table_name=route.tmp_db_table_name,
+                    geotype=route.geometry_type.upper(),
+                ),
+                pass_value=1,
+            )
+        )
+
+    checks = count_checks + colname_checks + geo_checks
+    multi_check = PostgresMultiCheckOperator(
+        task_id="multi_check", checks=checks, params=make_params(checks)
+    )
```

Comment:
Waarom de `make_params` nog apart hier doen, als je deze ook in de `PostgresMultiCheckOperator.__init__` kan genereren? Dan wordt de code ook wat leesbaarder van:

```suggestion
    multi_check = PostgresMultiCheckOperator(
        task_id="multi_check", checks=[count_checks + colname_checks + geo_checks]
    )
```

Wel hele coole class: PostgresMultiCheckOperator! 😄

Reply:
Ja, goed punt. Begon eerst met die params. Toen bedacht ik dat ik daar wel op wilde checken. Maar idd, nu kan het net zo goed helemaal in de constructor :-)